### DISCUSSION TITLE
Convert existing vods

### DISCUSSION THREAD

**Discussion by noraemsu** [Zibbp/ganymede#396] [Ideas]
Hi, just a quick question.
Have you thought about adding an option to convert an existing vod file from mp4 to hls playlist. Just like the the `Convert to HLS` setting but a button to either convert all existing mp4 files or individual streams?
Thanks!

**Comment by Zibbp** [comment]
Hi,
I don't have a plan to implement something like this. It's unlikely it will be used by many people. You can convert the video to HLS yourself if you want with ffmpeg. The specific command I use can be found in the code. Just make sure that the files are in the hls directory, looking at the files of an archived vod with the HLS setting will help. https://github.com/Zibbp/ganymede/blob/a20bc20f4b85af91e384138d8e92104f11ccfe48/internal/exec/exec.go#L180
